libs/create-zephyr-apps/src/templates.ts

```diff
@@ -68,4 +68,9 @@ export const Templates: { name: string; label: string; hint: string }[] = [
     label: 'Turbo + Rspack + Module Federation',
     hint: 'A monorepo using Turborepo, React, and Rspack as the bundler.',
   },
+  {
+    name: 'rspress-ssg',
```

Comment:
name is the folder name of zephyr-examples.

Please choose either `ssg-rspress` or `rspress-ssg` across both examples and here.

Reply:
I though was poiting to the packages example. Renaming in both repo for rspress-ssg

Reply:
@arthurfiorette Fixed the folder name inconsistency along a full disclosure on @VladyslavZinchenko pointing as well: https://github.com/ZephyrCloudIO/zephyr-examples/pull/102#issuecomment-2840998205